vllm/config/speculative.py

```diff
@@ -425,6 +461,12 @@ def __post_init__(self):
                         self.target_parallel_config,
                         self.draft_tensor_parallel_size))
 
+        if self.use_ngram() and not self.disable_padded_drafter_batch:
+            logger.warning(
+                "padded_drafter_batch has to be disabled with ngram. "
+                "Setting it disable_padded_drafter_batch to True.")
+            self.disable_padded_drafter_batch = True
```

Comment:
@benchislett - jfyi, padded_drafter_batch has been disabled by default for nrgam and ngram-eagle.